airbyte-integrations/connectors/source-harvest/source_harvest/streams.py

```diff
@@ -201,6 +201,13 @@ class EstimateItemCategories(IncrementalHarvestStream):
     Docs: https://help.getharvest.com/api-v2/estimates-api/estimates/estimate-item-categories/
     """
 
+    def read_records(self, **kwargs) -> Iterable[Mapping[str, Any]]:
```

Comment:
Can other streams face the same issue? I suggest you move this logic to the base class